spynnaker/pyNN/external_devices_models/munich_spinnaker_link_motor_device.py

```diff
@@ -259,4 +90,13 @@ def dependent_vertices(self):
     def edge_partition_identifiers_for_dependent_vertex(self, vertex):
         """ Return the dependent edge identifier
         """
-        return [MOTOR_PARTITION_ID]
+        return [self.machine_vertex.MOTOR_PARTITION_ID]
+
+    @overrides(AbstractProvidesOutgoingPartitionConstraints.
```

Comment:
shouldnt this be moved to the machine vertex?

Reply:
yes but out of scope of this PR.

 AbstractSpinnakerBase._add_dependent_verts_and_edges_for_application_graph

Check the appgraph not the machnegraph

Reply:
but the process outgoing edge partition fec function will check the machine vertex first